### PR TITLE
Fix casing

### PR DESCRIPTION
**Pull request by chcosta**
Fixes https://github.com/dotnet/core-eng/issues/7898#issuecomment-533172169

The only change I made to this was https://github.com/dotnet/arcade/commit/5b50ec4c80e3240d7195306952c4b451c16abf1d.  The casing got flipped and stage names are case dependent (sadly).

Validating with https://dev.azure.com/dnceng/internal/_build/results?buildId=360209&view=results

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -16,184 +16,174 @@ pr:
...
+- template: eng/common-variables.yml
 
 resources:
   containers:
   - container: LinuxContainer
     image: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
 
 stages:
-- stage: build
+- stage: Build
```
Comment (riarenas) [inline azure-pipelines.yml]: Should this be lowercased as well?

Comment (riarenas) [reply azure-pipelines.yml]: And the validate sdk stage that depends on this?

Comment (riarenas) [reply azure-pipelines.yml]: (and test)

Comment (chcosta) [reply azure-pipelines.yml]: A validation job for Arcade would catch this, but yes,  "Build" should be lower-cased as well.  

Validate and validate sdk and such are all defined in the templates and just need to match.

Comment (chcosta) [reply azure-pipelines.yml]: Oh, I see what you're referring to.  the stages in azure-pipelines got case changed.  Ack.

**Pull request merged**